qml/keys/AnnotatedKey.qml

```diff
@@ -202,7 +202,7 @@ Item {
         anchors.bottom: buttonImage.top
         width: key.width + Units.gu(UI.magnifierHorizontalPadding)
         height: key.height + Units.gu(UI.magnifierVerticalPadding)
-        text: keyLabel.text
+        text: UI.currentShiftState === "SHIFTED" ? annotationLabel.text : keyLabel.text
```

Comment:
This is incorrect is currentShitState === "CAPSLOCK"; better use valueToSubmit, has it necessarily matches what the user types